README.md

```diff
@@ -39,6 +39,22 @@ to
 image: ghcr.io/arm64-compat/apache/druid:25.0.0
 ```
 
+## Git Submodule upgrade
+
+Process to upgrade submodule:
+
+    1.- git checkout -b 10-upgrade-to-3000
+    2.- git submodule init
+    3.- git submodule update --remote druid
+    4.- git submodule foreach --recursive 'git fetch --tags'
+    5.- cd druid
+    6.- git checkout <tag> (e.g. git checkout druid-30.0.0)
+    7.- cd ..
+    8.- git add druid
+    9.- git submodule status 
+    10.- git commit -m "Upgrade to 30.0.0" 
+    11.- git push -u origin 10-upgrade-to-3000
+
```

Comment:
Feel free to update if wrong or missed something, but thats what I followed to upgrade it.